### ISSUE TITLE
Vue in dev mode

### ISSUE THREAD

**Issue by shiftux** [resoai/TileBoard#580]
Hey,

My setup is as follows: I'm running Docker on a RPi4 which hosts my containerized Home-Assistant and the MQTT server, etc. The HA config (and therefore the tileboard config) is mounted to the container. Installed the latest version of tileboard grabbing a release and unzipped that into the HA config. 

Inspeciting the tileboard site in Chrome I get a message about Vue being run in dev mode: 
`You are running Vue in development mode.
Make sure to turn on production mode when deploying for production.` 

Issue https://github.com/resoai/TileBoard/issues/451 suggests to run `npm run build:watch` to switch to production mode. Firstly npm isn't available on the HA container and if I were to install it there, it would mean that whenever I (re)start the HA container I'd have to go into the container manually and run npm manually?
Is there no other way, like an environment variable? or am I missing something here?

thanks!

HA image : homeassistant/raspberrypi4-64-homeassistant:0.110.4 
tileboard release: v2.1.2 (latest)

**Comment by rchl** [comment]
If you are grabbing `TileBoard.zip` from the Releases page then it's already compiled and built. You don't have to do anything to it besides extracting and creating the config file.

TileBoard doesn't use Vue so not sure what are you opening exactly...

**Comment by shiftux** [comment]
thanks for your reply @rchl  

i had a tileboard instance running previously. so trying to resolve the dev move issue I figured i'd migrate to a new release. 
I renamed my old `www` folder unzipped the new release and put it into `www/tileboard`, then I moved my old config.js file into there (as well as the refactored additional files) and restarted HA. 

For reference I'm attaching my current config file. 

regards
[config.js.zip](https://github.com/resoai/TileBoard/files/5781804/config.js.zip)
